This patch was created by:
* editing MODULE.bazel
* running a debug build (which fails)
* running `REPIN=1 bazel run @maven//:pin` within the debug environment
---
diff --git a/MODULE.bazel b/MODULE.bazel
index c83f8ab..c4f06e8 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -225,7 +235,7 @@ maven.install(
     ],
     boms = [
         "io.opentelemetry:opentelemetry-bom:1.46.0",
-        "io.netty:netty-bom:4.1.117.Final",
+        "io.netty:netty-bom:4.1.118.Final",
         "org.junit:junit-bom:5.11.4",
     ],
     excluded_artifacts = [
diff --git a/java/maven_install.json b/java/maven_install.json
index 9f5b646..fa60c1a 100644
--- a/java/maven_install.json
+++ b/java/maven_install.json
@@ -1,7 +1,7 @@
 {
   "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": "THERE_IS_NO_DATA_ONLY_ZUUL",
-  "__INPUT_ARTIFACTS_HASH": -83251177,
-  "__RESOLVED_ARTIFACTS_HASH": 159666128,
+  "__INPUT_ARTIFACTS_HASH": -1046694922,
+  "__RESOLVED_ARTIFACTS_HASH": -154146669,
   "artifacts": {
     "com.beust:jcommander": {
       "shasums": {
@@ -235,94 +235,94 @@
     },
     "io.netty:netty-buffer": {
       "shasums": {
-        "jar": "7d01bf0334ffa3ab55f3828bb50188bf9969774a23791a3df1905e53bfeea8f5",
-        "sources": "918f7561293bb6212646e7cb417b43ed7a638a29f3e08a21f7c540fd2757788e"
+        "jar": "0eea4e8666a9636a28722661d8ba5fa8564477e75fec6dd2ff3e324e361f8b3c",
+        "sources": "554d9b4e1211c0fe7e6703d8449755b226b48eea680c95fbcc246207c9f4835e"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-codec": {
       "shasums": {
-        "jar": "0b93f01cff5bc2a64af829ca01c826e8e861a03b96a871fa2a240fa25681df78",
-        "sources": "008eec001d04023dc6becd6b82a6848c10fbd6decfed5e6efeca46fe23532d40"
+        "jar": "4abd215fd1ed7ce86509d169cc9cbede5042176c265a79b3b70602b017226c3f",
+        "sources": "b1bdf9c511810a1f0a5485e1d652e0883f35118224d4dc21ef7edd4c822a477d"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-codec-dns": {
       "shasums": {
-        "jar": "1672af84022aaba02aaf8b4bdf76c9fb02b93983002e76f6a392b477b326479c",
-        "sources": "d1ad3938e8b2586058dfc8f67b334c90093f38cfbaed8d981a06c5d3f4b94505"
+        "jar": "e115e42ca1e3cc8d85e3a632d8faa102d18c0ebc1fa4511af30bec79f8c147d4",
+        "sources": "ede3ae4ee1e0476c00da1e04927cc2bdc3ca84f9b506378a3b82e333b7053d6a"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-codec-http": {
       "shasums": {
-        "jar": "a7e19945a6baa86ab285383a0bda632eae7703a889da1d20ea4eaad4c996cc21",
-        "sources": "ac6d1a9e7897235a149d5050f0970b681bf979b48489d67ba0125f239d1672bb"
+        "jar": "09822d785e9a794838031ddd5346cf419b30c036a981c2e277a062bea884174b",
+        "sources": "637f46d46c300cdcb097aab6ed89013fca85bb89b798763fc962e09a3b91cef2"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-codec-http2": {
       "shasums": {
-        "jar": "10e60a0c2a86568720f1d9b50ed6e33728d12e7ccb1cc961828f66cc6a73aa19",
-        "sources": "58d002225ca0843d9a884bd61b816bc61ed8c067fea767f7eaf1dbcb2decc4bd"
+        "jar": "68da0b1a34dceb00a6f9f6f788fb2f6b7b9e4adba8c70658ac2bd7eb898b97ae",
+        "sources": "3850183e9d162944439b78dccbc9907cf06d66abc8116f56ebc03cb3256042a7"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-codec-socks": {
       "shasums": {
-        "jar": "870bec5ad7650d59bcf610ea8461dc7a4a6dd8f2538cbc024dd0f39b45cad2d7",
-        "sources": "08e89f4cb30df957ddf71c55ed070b15d77ee1c7e8625afd7577cd6e89539886"
+        "jar": "094465e3cfb3aef0fca38ed82b801f53a6c8be7ae1f83ab0c1b2e8ece2586840",
+        "sources": "6a0c8c5394eb77afc55ea417a853f49bd7a54036b2fd6825a80587c244053835"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-common": {
       "shasums": {
-        "jar": "5b4e125a341566b24d5030479da475b20ea1f6145bf7b3c3d69fe537088bb4c1",
-        "sources": "a981e69ef6c23db2e2ba1cccc567dae50480bda46415f564acfe033fc923d153"
+        "jar": "65cce901ecf0f9d6591cc7750772614ab401a84415dc9aec9da4d046f0f9a77c",
+        "sources": "e3c2314093e094f29671c91618fd8e9cb629dcdd7c879f3b3ee4566e53568feb"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-handler": {
       "shasums": {
-        "jar": "f3272149099883cf98ad8a2a82a08111a78d17e604aebc094c80592a730fcac5",
-        "sources": "8a18ddaffbc8f30c7f06b135f06f9b418c65717f3f8c19da6e3726dfd3d3a185"
+        "jar": "26e3f8a5e859fd62cf3c13dc6d75e4e18879f000a5d0ad7f58f8679675d23dae",
+        "sources": "650bd96999c937c8541e11fa9d9684261f8e14829737fcc2705afeeec928ef15"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-handler-proxy": {
       "shasums": {
-        "jar": "6bd8977daec5e30d3f9d286665e4dd20260eb0d3f87b71a492a62f5392daf871",
-        "sources": "c442ea6a6aa1c28729cd3953ee5680ffa01e288673b64ad24bed9dc57872ec01"
+        "jar": "fef926126f44c668968dd3e2389c2552981d452e6dfc23b1f9bd03db92c21f96",
+        "sources": "c83d6a07bdcbd2abd1375cc909c07db9a5e7e48f931c3f3434b2321953325395"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-resolver": {
       "shasums": {
-        "jar": "0f5f7220d34d32751cc624bc9c47afdde7392f0d9551569c90a568309015d796",
-        "sources": "2716e7689a71211ec2e6fb443f44d3ce82e218cc52f25bd541531e7883234d35"
+        "jar": "3170c225972c18b6850d28add60db15bb28d83c4e3d5b686ca220e0bd7273c8a",
+        "sources": "bdcc9f75ec429c025a1a28e7fafa45401b057f19823af8b594c0cf58273c1006"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-resolver-dns": {
       "shasums": {
-        "jar": "77e21a62c155d190335002ecc37bbe9c5f54532064121871d3eeb82cded65239",
-        "sources": "a00f2c50dd19655916e3d7c0cefe783fdd9c20c9f463aa33d0613e4f695cddf3"
+        "jar": "c0e0fdaffaba849e3145b2b96288fc8fc6f3b2a623cf72aaba708288348e4938",
+        "sources": "034920a2447a06f1f9719704b03f3dfd79dd70412c6757eccead74ec1b6ba882"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-transport": {
       "shasums": {
-        "jar": "c07aac2ea55aa42c34fc6b53fbee661e0de077f45483133b6ca297c70a67a44a",
-        "sources": "528859630b9e07e317fc912dd75c6b835cd6169291d9658d8e93f9a664f344f8"
+        "jar": "ab3751e717daef9c8d91e4d74728a48730bd8530b72e2466b222b2ea3fb07db9",
+        "sources": "0ed9ee6aad925eb723b491adc5715d399be2207d8e0ef6d5741e61751d5074cf"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.netty:netty-transport-native-unix-common": {
       "shasums": {
-        "jar": "97a8b5848bde9d12e80a3ddeb1f3d93bba6d123181aa71524eff2ab8458431ec",
-        "sources": "889b35bc30eed91e77920d1a70e770a77b9468100dfc240ea01c04af721ba1ec"
+        "jar": "69b16793d7b41ea76a762bd2bd144fc4f7c39c156a7a59ebf69baeb560fb10b7",
+        "sources": "ff5b22a1e0b5c72fd37770f035e69a5a69cdacb3dd4ac0601d3be6c1b25e8ba0"
       },
-      "version": "4.1.117.Final"
+      "version": "4.1.118.Final"
     },
     "io.opentelemetry.semconv:opentelemetry-semconv": {
       "shasums": {
